tst/testmanuals.g

```diff
@@ -0,0 +1,97 @@
+#############################################################################
+##
+#W  testmanuals.g
+##
+##  <#GAPDoc Label="[1]{teststandard.g}">
+##  <#/GAPDoc>
+##
+
+# This code extracts the examples from manuals chapter-wise and
+# stores this in a workspace.
+
+Print("Extracting manual examples...\n");
+Read(Filename(DirectoriesLibrary("doc/ref"), "makedocreldata.g"));
+
+GAPInfo.ManualDataRef.pathtodoc := DirectoriesLibrary("doc/ref");
```

Comment:
Of course keeping the `Read` as it is can be convenient when debugging this file, as it allows one to copy&paste that line into a GAP session.

So an alternative would be to get rid of `GAPInfo.ManualDataRef.pathtodoc` (or is it used outside of this script). There is only one place here using it...